benchmark/single-source/ArrayOfGenericPOD.swift

```diff
@@ -59,7 +59,7 @@ func genStructArray() {
 
 @inline(never)
 public func run_ArrayOfGenericPOD(_ N: Int) {
```

Comment:
This one is 50% overhead according to my measurements. I guess some `blackhole`s are needed in there, too?

Reply:
I didn't see any problems here, but it's a good idea to add blackHoles here.